core/src/test/java/org/wso2/testgrid/core/command/GenerateTestPlanCommandTest.java

```diff
@@ -79,6 +91,14 @@ public void init() throws Exception {
         int random = new Random().nextInt(100);
         product.setName("wso2-" + random);
 
+        this.testPlan = new TestPlan();
+        testPlan.setId("abc");
```

Comment:
Shall we use consistent Test plan IDs? I see that we used TP_1 in the other test case.

Reply:
@kasunbg : Haven't add a constant since it has not been used more than once. Do we need to?